fcrepo-server/src/test/java/org/fcrepo/server/storage/DefaultExternalContentManagerTest.java

```diff
@@ -110,8 +113,8 @@ private void mockResponseFor(String httpMethod, int status) {
         when(mockResponse.getResponseHeaderValue(HttpHeaders.CONTENT_LENGTH, "-1")).thenReturn(cLen);
         when(mockResponse.getResponseHeaders()).thenReturn(new Header[0]);
         when(mockResponse.getStatusCode()).thenReturn(status);
-        mockStatic(DOTranslationUtility.class);
-        when(DOTranslationUtility.makeAbsoluteURLs(TEST_URL)).thenReturn(TEST_URL);
```

Comment:
With this stuff gone, can we take the `DOTranslationUtility.class` out of the PowerMock annotation above?